tools/testsys/src/aws_resources.rs

```diff
@@ -170,10 +238,22 @@ impl AwsK8s {
     }
 
     /// Labels help filter test results with `testsys status`.
-    fn labels(&self) -> BTreeMap<String, String> {
+    fn labels<S1, S2>(
+        &self,
+        cluster_name: S1,
+        testsys_type: S2,
+        test_type: &TestType,
+    ) -> BTreeMap<String, String>
+    where
+        S1: Into<String>,
+        S2: Into<String>,
+    {
         btreemap! {
             "testsys/arch".to_string() => self.arch.to_string(),
             "testsys/variant".to_string() => self.variant.to_string(),
+            "testsys/cluster".to_string() => cluster_name.into(),
+            "testsys/type".to_string() => testsys_type.into(),
+            "testsys/test".to_string() => test_type.to_string(),
```

Comment:
It occurs to me, relating to this https://github.com/bottlerocket-os/bottlerocket-test-system/issues/628, that we use `testsys` here as if we "own" that word, but in the afore mentioned issue we are contemplating making `testsys` the namespace of the more general testing system. I guess it's fine and reflects the journey we have taken...